packages/ckb-sdk-core/src/generateRawTransaction.ts

```diff
@@ -51,9 +52,9 @@ const generateRawTransaction = ({
   assertToBeHexStringOrBigint(fee)
   assertToBeHexStringOrBigint(capacityThreshold)
   assertToBeHexStringOrBigint(changeThreshold)
-  const targetFee = BigInt(fee)
-  const minCapacity = BigInt(capacityThreshold)
-  const minChange = BigInt(changeThreshold)
+  const targetFee = JSBI.BigInt(`${fee}`)
+  const minCapacity = JSBI.BigInt(`${capacityThreshold}`)
+  const minChange = JSBI.BigInt(`${changeThreshold}`)
```

Comment:
[`${fee}`] can be replaced by [fee]

Reply:
> `${fee}` can be replaced by fee

`fee` could be a bigint and `JSBI.BigInt` doesn't accept it.